pkgs/shells/rssh/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, openssh, rsync, cvs }:
+
+stdenv.mkDerivation rec {
+  name = "rssh-${version}";
+  version = "2.3.4";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/rssh/rssh/${version}/${name}.tar.gz";
+    sha256 = "f30c6a760918a0ed39cf9e49a49a76cb309d7ef1c25a66e77a41e2b1d0b40cd9";
+  };
+
+  patches = [
+    ./0001-fail-logging.patch
+    ./0002-info-to-debug.patch
+    ./0003-man-page-spelling.patch
+    ./0004-mkchroot.patch
+    ./0005-mkchroot-arch.patch
+    ./0006-mkchroot-symlink.patch
+    ./0007-destdir.patch
+    ./0008-rsync-protocol.patch
+    ./fix-config-path.patch
```

Comment:
is it possible to source the patches from AUR directly ? there are quite a few patches and we are trying to avoid nixpkgs becoming too big